apollo_integration_tests: generate empty contract declare tx (#7718 part 2)

itamar-starkware opened the pull request:
We will use the empty contract in the following PR's so we declare it now.

ArniStarkware reviewed (changes requested):
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/9194)* status: 0 of 2 files reviewed, 1 unresolved discussion (waiting on @ayeletstarkware)

___
*[`crates/apollo_integration_tests/tests/test_custom_syscall_invoke_txs.rs` line 46 at r2](https://reviewable.io/reviews/starkware-libs/sequencer/9194#-O_CI-v6BRvhy6GtZUXk:-O_CI-v6BRvhy6GtZUXl:b-mq4fpq) ([raw file](https://github.com/starkware-libs/sequencer/blob/8b5fe2fee792a2404e75eacaf4214b5ebf0d46d1/crates/apollo_integration_tests/tests/test_custom_syscall_invoke_txs.rs#L46)):*
> ```XML
> /// Creates a set of transactions that test the Cairo 1.0 syscall functionality.
> /// The transactions are taken from starkware repo.
> fn create_cairo_1_syscall_test_txs(
> ```

I remember we talked about the name of this function. So *Now* the name is no longer relevant.
This is no longer just for syscalls.
Also - the next function has the word `invoke` in the name of the function.

Should this sepcific scenario be included in a different test?



_Code quote:_
```XML
fn create_cairo_1_syscall_test_txs(
```


<!-- Sent from Reviewable.io -->

itamar-starkware commented:
*[`crates/apollo_integration_tests/tests/test_custom_cairo1_txs.rs` line 120 at r3](https://reviewable.io/reviews/starkware-libs/sequencer/9194#-O_Gf0b-ATtu-Qwh3tv_:-O_Gh9mUBCyxIpdeLFuX:b-9s4b0p) ([raw file](https://github.com/starkware-libs/sequencer/blob/e699a16842b5f4f68854a656be84ce25c8723a11/crates/apollo_integration_tests/tests/test_custom_cairo1_txs.rs#L120)):*
<details><summary><i>Previously, ayeletstarkware (Ayelet Zilber) wrote…</i></summary><blockquote>

is the empty contract written in cairo1/cairo0? if so, we need a compiled class hash for both of them/ declare the cairo version in the line above as in the function above.
</blockquote></details>

Cairo1, I changed the declareation as you suggested to make it more clear.
I changed the constant name to identify it so it will be clear it cairo 1.


<!-- Sent from Reviewable.io -->

ayeletstarkware reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> @ayeletstarkware reviewed 1 of 2 files at r1, 2 of 2 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/sequencer/9194#-:-O_GrPUvDxyN_4oF_Itg:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @ArniStarkware)

___
*[`crates/mempool_test_utils/src/lib.rs` line 6 at r4](https://reviewable.io/reviews/starkware-libs/sequencer/9194#-O_GrE7q1ymlHy5bERdt:-O_GrE7q1ymlHy5bERdu:b-irm4m1) ([raw file](https://github.com/starkware-libs/sequencer/blob/e213cb68c5c247ca384df1e1c9f739b363cee37a/crates/mempool_test_utils/src/lib.rs#L6)):*
> ```XML
> pub const CONTRACT_CLASS_FILE: &str = "contract_class.json";
> pub const COMPILED_CLASS_HASH_OF_CONTRACT_CLASS: &str =
>     "0x24d8d75cba029fa1896dd4d9424df66f8b279b058b4c6f3245b369c78c5d156";
> ```

non blocking - consider adding cairo1/cairo0 to these constants as well (in a different PR).



_Code quote:_
```XML
pub const CONTRACT_CLASS_FILE: &str = "contract_class.json";
pub const COMPILED_CLASS_HASH_OF_CONTRACT_CLASS: &str =
    "0x24d8d75cba029fa1896dd4d9424df66f8b279b058b4c6f3245b369c78c5d156";
```


<!-- Sent from Reviewable.io -->
